sklearn/metrics/tests/test_common.py

```diff
@@ -556,7 +555,7 @@ def test_not_symmetric_metric(name):
     metric = ALL_METRICS[name]
 
     # use context manager to supply custom error message
-    with assert_raises(AssertionError) as cm:
+    with pytest.raises(AssertionError) as cm:
```

Comment:
This will not work as intended with pytest.

I think that we want something like this instead

```python
with pytest.raises(AssertionError):
    assert_array_equal(metric(y_true, y_pred), metric(y_pred, y_true))
    raise ValueError("%s seems to be symmetric:" %name)
```

```

Reply:
@glemaitre Thanks for pointing that out! Will make the required change.